dashboard/app/models/foorm/simple_survey_form.rb

```diff
@@ -22,4 +22,13 @@ class Foorm::SimpleSurveyForm < ApplicationRecord
     'survey_data',
     'allow_multiple_submissions'
   ]
+
+  def self.find_most_recent_form_for_path(path)
+    where(path: path).last
```

Comment:
i think this might technically work because it looks like the default sorting is by id, but maybe add `.order(created_at: :asc)`